multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -159,45 +159,25 @@ coreos:
         ExecStartPre=/usr/bin/curl http://127.0.0.1:8080/version
         ExecStart=/opt/bin/install-kube-system
 
-{{ if .UseCalico }}
-    - name: install-calico-system.service
-      command: start
-      content: |
-        [Unit]
-        Requires=kubelet.service docker.service
-        After=kubelet.service docker.service
-
-        [Service]
-        Type=simple
-        StartLimitInterval=0
-        Restart=on-failure
-        ExecStartPre=/usr/bin/curl http://127.0.0.1:8080/version
-        ExecStart=/opt/bin/install-calico-system
-{{ end }}
-
 write_files:
+  - path: /opt/bin/kubectl
```

Comment:
Every time we run this it is going to create a new container - I'm not sure we want to expose this in a generic way on the hosts (Ideally we just want to use it once on first boot). In normal operations an admin shouldn't need to execute kubectl on the host itself.

If we do want to provide this as a generic tool, we probably need to add more into this wrapper like self-cleanup after execution. See flannel-wrapper / flannel.service wrappers @steveeJ has been working on: https://github.com/coreos/coreos-overlay/pull/2094